src/js/controllers/import.js

```diff
@@ -25,15 +27,15 @@ angular.module('copayApp.controllers').controller('importController',
 
       if ($scope.isChromeApp) {
```

Comment:
In join.js you're comparing agains `walletService.externalSource.ledger.supported` and `walletService.externalSource.trezor.supported` I'm just wondering if you should use the same method in create.js.